lean4-fringe.el

```diff
@@ -75,7 +75,7 @@
 (defvar-local lean4-fringe-data nil)
 
 (defun lean4-fringe-update-progress-overlays ()
-  "Update 'processing' bars in the current buffer."
+  "Update `processing' bars in the current buffer."
```

Comment:
Backquotes inside documentation strings should be used for Emacs Lisp symbols. These backquoted strings will be annotated in help buffers, which is probably not what you want.

According to GNU Emacs Lisp Reference Manual corresponding to Emacs version 30.0.50, a correct solution here seems to be `’processing’` (not an apostrophe, but a right single quotation mark on both sides):

> For example, the documentation string "Alias for
'foo'." is probably not what you want, as it can display as "Alias for
’foo’.", an unusual style in English.

Reply:
The pair (grave accent, apostrophe) in a docstring is translated to (left single quote, right single quote) for display. It's dumb, but here we are, for historical reasons.

Of the 6 instances (in 5 docstrings) of incorrect quoting using a pair of apostrophes:

* one is scare quotes (`lean4-fringe-update-progress-overlays`, "Update 'processing' bars in the current buffer."); these are better omitted

* two are the filename "lakefile.lean" (see `lean4-lake-find-dir-in`, `lean4-lake-find-dir`); prefer double quotes, to remind the reader of a string literal

* one is shell syntax (lean4-lake-build, "Call lake build."): I should have left this as is

* the last two (see lean4-compile-string, "Command to run EXE-NAME with extra ARGS and FILE-NAME. If LAKE-NAME is nonempty, then prepend 'LAKE-NAME env' to the command 'EXE-NAME ARGS FILE-NAME'."): really, who cares? Let's say double quotes

Do you agree with those suggestions?

Reply:
Thanks. Looks good to me. I am lazy, so I fall back to double quotes in most situations unless for Emacs Lisp symbols.  You seem to have paid enough attention to the issue here.

Reply:
Committed. Should I squash?

Reply:
I am not a maintainer of this repository, though I have committed some amount of changes to help them get on board MELPA. Please ask the maintainers.

Reply:
OK. Thank you very much for your help refining this PR. The maintainers will doubtless chime in if and when.